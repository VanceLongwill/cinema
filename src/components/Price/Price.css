.Price {
  font-weight: 700;
  padding: var(--spacing-sm);
  background: var(--secondary-dark-color);
}
