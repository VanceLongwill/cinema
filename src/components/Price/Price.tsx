import React from "react";

import "./Price.css";

export const Price: React.FunctionComponent<{ value: string }> = ({
  value
}) => <span className="Price">{value}</span>;
