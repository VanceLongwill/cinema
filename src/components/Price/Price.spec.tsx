import * as React from "react"

import { mount } from "enzyme"

import { Price } from "./Price"

describe("Price", () => {
  it("should render without errors", () => {
    expect(mount(<Price />)).toMatchSnapshot()
  })
})
