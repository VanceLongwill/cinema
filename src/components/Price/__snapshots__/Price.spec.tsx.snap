// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price should render without errors 1`] = `
<Price>
  <span
    className="Price"
  />
</Price>
`;
